src/main/java/racinggame/domain/Winner.java

```diff
@@ -0,0 +1,46 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Winner {
+
+    private final List<Car> cars;
+
+    public Winner(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<String> getWinners() {
+        return getWinnerByPosition(getWinnerPosition());
+    }
+
+    private List<String> getWinnerByPosition(int winnerPosition) {
+        List<String> winnerList = new ArrayList<>();
+        for (Car car : cars) {
+            winnerList.add(getWinnerCarName(winnerPosition, car));
+        }
+        winnerList.removeIf(Objects::isNull);
+        return winnerList;
```

Comment:
Stream API를 좀 더 활용해보면 어떨까요? 
```suggestion
        cars.filter(car -> car.equalPosition(winnerPosition)).map(Car::getName).collect(Collectors.toList());
```